### PR TITLE
Update Pull Request Templates

### PR DESCRIPTION
**Pull request by marcfreiheit**
[//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)

**Related Task**: #122 

**Description**

[//]: # (Describe the new feature and how you've implemented it)

Updates the Pull Request templates to include a technical discussion. We have to discuss how to provide the templates for all developers.

**Steps to reproduce**

[//]: # (Please provide a description about how to test the new functionality. Don't forget to name user accounts required to test the feature.)

Pull Request templates can't be selected visually in GitHub. In order to load a Pull Request template, you have to add an url parameter while creating the pull request:

`?template=task.md|bug.md|user-story.md`

In my case, the url looked like: `https://github.com/hpi-swt2/vm-portal/compare/50/login/issue-templates?expand=1&template=task.md`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 1]
```diff
@@ -4,10 +4,18 @@ about: This template should be used for all user stories
 
 ---
 
-**Beschreibung**
+[//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)
 
-**Akzeptanzkriterien**
+**User Story**
 
-**Priorität**
```
Comment (MkuuWaUjinga) [inline .github/ISSUE_TEMPLATE/user-story.md]: Why did you remove the priority estimation? Should we use labels instead?

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+**Summary**
```
Comment (MkuuWaUjinga) [inline .github/ISSUE_TEMPLATE/bug.md]: It would be helpful to state here as well that [//]-blocks do not have to be removed e.g. [//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)

File: .github/PULL_REQUEST_TEMPLATE/bug.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+**Related Issue:** GH-
```
Comment (MkuuWaUjinga) [inline .github/PULL_REQUEST_TEMPLATE/bug.md]: It would be helpful to state here as well that [//]-blocks do not have to be removed e.g. [//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)

File: .github/PULL_REQUEST_TEMPLATE/task.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+
+[//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)
+
+**Related Task**: #
```
Comment (MkuuWaUjinga) [inline .github/PULL_REQUEST_TEMPLATE/task.md]: Add something like "Please provide the id of the issue of the task you've solved. This should be used in order to get information about the task itself."

File: .github/PULL_REQUEST_TEMPLATE/user-story.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+
+[//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)
+
+**Related User Story**: #
```
Comment (MkuuWaUjinga) [inline .github/PULL_REQUEST_TEMPLATE/user-story.md]: Add something like "Please provide the id of the user story of the task you implemented. This should be used in order to get information about the user story itself."

**Comment by arne-z** [comment]
It's been 2 months since this saw any comments/changes so I will close it for now. If anyone still needs it, feel free to complain/reopen.
